README.md

```diff
@@ -17,4 +17,85 @@ Sample output:
     "percentChange":-90.54688681639148
   }
 }
-```
\ No newline at end of file
+```
+
+### `/public/trades`
+Historical trades
+Return historical trades executed. Trades returned in ascending order. Option available to query historical trades based on tradeID, default is
+to return latest 100 trades
+
+#### Params
+
+|Name        | Type                 | Example                                             | Description
+|------------|----------------------|-----------------------------------------------------|----------------------------
+|pair        | String               | ETH_EOS                                             | 
+|fromId      | Number               | 123                                                 | query by tradeID, display trades after this ID inclusive (optional, default to return latest trades)
+|limit       | Integer              | 200                                                 | number of trades returned (optional, default to 100)
+|order       | String (asc or desc) | desc                                                | order which items returned in (default to desc)
+
+#### Sample input: 
+
+`GET /public/trades?pair=ETH_EOS&limit=20&fromId=123` 
+
+#### Sample output:
+```
+[
+   {
+      "id":123,
+      "timestamp":1520797156,
+      "createdAt":"2018-03-11T19:39:16.967Z",
+      "amount":0.01,
+      "price":2,
+      "type":"sell",
+      "tokenGet":"ETH",
+      "tokenGive":"EOS"
+   },
+   {
+      "id":123,
+      "timestamp":1520797166,
+      "createdAt":"2018-03-11T19:39:26.011Z",
+      "amount":0.01,
+      "price":2,
+      "type":"sell",
+      "tokenGet":{"symbol": "ETH", "decimals": 3},
+      "tokenGive":{"symbol": "EOS", "decimals": 3}
```

Comment:
Do we really need decimals here? I don't think so, we return proper prices already. So maybe just return "tokenGet": "ETH"? Are there any reasons to return decimals?